analyzer/consensus/consensus.go

```diff
@@ -720,10 +720,11 @@ func (m *Main) queueEscrows(batch *storage.QueryBatch, data *storage.StakingData
 					SET
 						escrow_balance_active = escrow_balance_active - $2,
 						escrow_balance_debonding = escrow_balance_debonding + $2
+						escrow_total_shares_debonding = escrow_total_shares_debonding + $2
 					WHERE address = $1;
 			`, chainID),
 				e.DebondingStart.Escrow.String(),
-				e.DebondingStart.DebondingShares.ToBigInt().Uint64(),
```

Comment:
We want the number of shares to be [reclaimed](https://github.com/oasisprotocol/oasis-core/blob/2b35f69b5a8fa3d9b8e1dd19123d4e3da72b5e38/go/consensus/tendermint/apps/staking/transactions.go#L475).